mutiny-core/src/redshift.rs

```diff
@@ -379,7 +379,7 @@ impl RedshiftManager for NodeManager {
 
             // make attempts to pay it
             match sending_node
-                .pay_invoice_with_timeout(&invoice, None, None)
+                .pay_invoice_with_timeout(&invoice, None, None, vec![])
```

Comment:
Could use a todo comment here mentioning that we should add labels for redshift.